resources/options.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Created by Mitchell on 10-5-2016.
+ * Contains necessary functions for the Settings page of Octopeer.
+ */
+
+// Helper function; simplifies the calls to specific document e-L-ements.
+function l(id) {
+    return document.getElementById(id)
```

Comment:
I used this so much I didn't even notice XD But indeed, jQuery is even already loaded into the page :)